### ISSUE TITLE
Unused lr_mul for the backbone?

### ISSUE THREAD

**Issue by GuHY777** [MalongTech/research-ms-loss#27]
https://github.com/msight-tech/research-ms-loss/blob/b68507d4e22d8a6d3d3c0e6c31be708f9dcd20ee/ret_benchmark/solver/build.py#L14

There is not "lr_mul" in torch.optim, so the backbone's learning rate will be the same as embedding layer?
